java/ql/automodel/src/AutomodelFrameworkModeCharacteristics.qll

```diff
@@ -24,9 +24,17 @@ newtype JavaRelatedLocationType =
   ClassDoc()
 
 newtype TFrameworkModeEndpoint =
-  TExplicitParameter(Parameter p) or
+  TExplicitParameter(Parameter p) { not p.getType() instanceof PrimitiveType } or
   TQualifier(Callable c) { not c instanceof Constructor } or
-  TReturnValue(Callable c) or
+  TReturnValue(Callable c) {
+    c instanceof Constructor
```

Comment:
I don't know if it matters, but in MaD constructors are modeled as `Argument[this]` instead of `ReturnValue`.

Reply:
I think that matters, but I'll move investigating and (if needed) fixing this to a follow-up PR, ok?

Reply:
Alternatively, we could drop support for constructor source models now and add it in back later. From existing modeling, it seems like we don't have a single constructor model for `remote` data in `lib/ext` currently.

Reply:
Let's see the candidates this generates and make a decision then, if you're OK with it.